apps/changelog/commands.py

```diff
@@ -0,0 +1,39 @@
+# Third Party (PyPI) Imports
+import click
+
+# HTK Imports
+from htk.apps.changelog.classes import (
+    ChangeLog,
+    LogEntry,
+)
+from htk.apps.changelog.utils import (
+    fetch_git_logs,
+    fetch_origin_url,
+)
+
+
+@click.command()
+@click.option(
+    '--slack-announce/--no-slack-announce',
+    default=False,
+    help='Announce latest changes to Slack',
+)
+def update_changelog_command(slack_announce):
+    origin_url = fetch_origin_url()
+    lines = fetch_git_logs()
+    log_entries = [LogEntry.from_line(origin_url, line) for line in lines]
+
+    changelog = ChangeLog(origin_url, log_entries)
+
+    changelog_file_name = './CHANGELOG.md'
```

Comment:
This should be customizable

Reply:
Done. Created a factory function to generate CLI function